src/app/setup/github-oauth.component.scss

```diff
@@ -1,6 +1,7 @@
-@import 'patternfly-sass/assets/stylesheets/patternfly/color-variables';
+@import '../../../node_modules/patternfly-sass/assets/stylesheets/patternfly/color-variables';
 
 .github-oauth {
+  background-color: #FFFFFF;
```

Comment:
Isn't there a patternfly color variable for this?

Reply:
Sure, can change this.